### ISSUE TITLE
[FEA] Support ProjectExec in LoRe framework

### ISSUE THREAD

**Issue by winningsix** [NVIDIA/spark-rapids#10862]
**Is your feature request related to a problem? Please describe.**
Support LoRe for project exec https://github.com/NVIDIA/spark-rapids/blob/188eb39a4b0a012f251d764176c466be555c6018/sql-plugin/src/main/scala/com/nvidia/spark/rapids/basicPhysicalOperators.scala#L350

**Comment by res-life** [comment]
PR: https://github.com/NVIDIA/spark-rapids/pull/10825

**Comment by jlowe** [comment]
@res-life has this been superceded by #10987?

**Comment by res-life** [comment]
Yes, superseded by https://github.com/NVIDIA/spark-rapids/issues/10987, close this.
